patches/components-payments-content-BUILD.gn.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/components/payments/content/BUILD.gn b/components/payments/content/BUILD.gn
+index fa171bba474a1b6ab3ff7f968959a0ffd3ef6f3c..28d42c4e024d3646cad840d6520fef58f671d7bb 100644
+--- a/components/payments/content/BUILD.gn
++++ b/components/payments/content/BUILD.gn
+@@ -3,6 +3,7 @@
+ # found in the LICENSE file.
+ 
+ import("//build/config/chromeos/ui_mode.gni")
++import("//brave/components/payments/content/brave_payment_sources.gni")
+ 
+ static_library("content") {
+   sources = [
+@@ -59,9 +60,12 @@ static_library("content") {
+     "service_worker_payment_app_factory.h",
+   ]
+ 
++  if (!is_android) { sources += brave_payment_sources }
++
+   deps = [
+     ":content_common",
+     ":utils",
++    "//brave/components/brave_rewards/common",
```

Comment:
why would you only be adding this? You need to include all the required deps in brave_payment_deps and do deps +=

Reply:
resolved